packages/fetchai/skills/confirmation_aw1/registration_db.py

```diff
@@ -85,6 +85,13 @@ def is_registered(self, address: str) -> bool:
         result = self._execute_single_sql(command, variables)
         return len(result) != 0
 
+    def get_all_registered(self) -> List[str]:
```

Comment:
check

Reply:
`result` is an instance of `List[Tuple[Any, ...]]`

Reply:
I think Any should be string no? It only returns strings afaik

Reply:
the return value might be `None` for nullable columns, so maybe `Optional[str]` is better.